desc/plotting.py

```diff
@@ -1390,6 +1509,17 @@ def plot_qs_error(
     ax : matplotlib.axes.Axes or ndarray of Axes
         Axes being plotted to.
 
+    Examples
+    --------
+
+    .. image:: ../../_static/images/plotting/plot_qs_error.png
+
+    .. code-block:: python
+
+        from desc.equilibrium import EquilibriaFamily
+        from desc.plotting import plot_qs_error
+        eq = EquilibriaFamily.load(load_from='../../examples/DESC/HELIOTRON_output.h5')[-1]
+        fig, ax = plot_qs_error(eq, helicity=(0, 0), log=False)
```

Comment:
Helicity of (M=0,N=0) is not physical and the resulting plot shows perfect Boozer QS. The helicity input should be changed to `helicity=(1, eq.NFP)` to target QH.

Reply:
That, combined with log=True makes it look a bit more reasonable. Though in the future it would be nice to implement the normalization by B^4 etc for plotting as well